.appveyor.yml

```diff
@@ -6,7 +6,13 @@ init:
   - ps: $PSVersionTable
 
 build_script:
-  - ps: ./bootstrap.ps1
-
-test_script:
-  - ps: chef exec inspec exec test/integration/**/*.rb --format documentation
+  - ps: . { (New-Object System.Net.WebClient).DownloadString('https://omnitruck.chef.io/install.ps1') } | iex; install -project 'chefdk' -version '0.17.17'
```

Comment:
Its basically the same, this is emulating the bootstrap script but locally, so closer to what will happen